### PR TITLE
RF-9: User Info CRUD

### PR DESCRIPTION
**Pull request by AgustinCarthery**
# [RF-9: User Info CRUD ](https://rutinesandfit.atlassian.net/browse/RF-9)

## Type of change

> - Feature (new feature of the server/application)

## Description

> - Added Routine and UserInfo schemas
> - Controllers and routes for CRUD operations

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -21,6 +21,11 @@ enum MuscleGroup {
   TRICEPS
 }
 
+enum Gender {
+  MALE
+  FEMALE
```
Comment (franco-marini) [inline prisma/schema.prisma:26]: We may add also, BINARY and PREFER_NOT_SPECIFY. WDYT?

**Review by franco-marini — COMMENTED** [review]
Just one little comment to improve

File: prisma/schema.prisma (hunk 1 above)
Comment (AgustinCarthery) [reply prisma/schema.prisma:26]: hahah will do 😈

**Review by franco-marini — APPROVED** [review]
🚀

**Pull request merged**